apps/stable_diffusion/src/utils/utils.py

```diff
@@ -253,11 +246,7 @@ def set_init_device_flags():
     ):
         args.use_tuned = False
 
-    elif "cuda" in args.device and get_cuda_sm_cc() not in [
-        "sm_80",
-        "sm_84",
-        "sm_86",
-    ]:
+    elif "cuda" in args.device and get_cuda_sm_cc() not in ["sm_80"]:
```

Comment:
what is this change for?

Reply:
We only want the cuda tuned configs to apply on A100, as we don't have tuning configs tested on other devices. We had seen problems applying the same config on sm_89. sm_89 has already been removed from the list.